src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js

```diff
@@ -0,0 +1,29 @@
+export default function httpErrorService( $state, $translate, notification) {
+    
+    return {
+    	handle_error: function(event, toState, toParams, fromState, fromParams, error){
+            switch (error.status) {
+            case 404:
+               this.handle_404_error();
+            case 403:
+               this.handle_403_error(error);
+            default:
+               this.handle_default_error(error);
+            }
+    	},
+    	handle_404_error: function(event,error){
+    		 $state.go('ma-404');
+             event.preventDefault();
```

Comment:
This is linked to the $stateChangeError UI router event, it stops the requested transition to happen